### ISSUE TITLE
issue installing

### ISSUE THREAD

**Issue by danhausman** [adriancable/homebridge-nest-native#3]
Here is the debug from trying to install it:

0 info it worked if it ends with ok
1 verbose cli [ '/usr/bin/node',
1 verbose cli   '/usr/bin/npm',
1 verbose cli   'install',
1 verbose cli   '-g',
1 verbose cli   'homebridge-nest-native' ]
2 info using npm@6.4.1
3 info using node@v8.16.0
4 verbose npm-session c50a387a09e464bc
5 silly install loadCurrentTree
6 silly install readGlobalPackageData
7 http fetch GET 404 https://registry.npmjs.org/homebridge-nest-native 593ms
8 silly fetchPackageMetaData error for homebridge-nest-native@latest 404 Not Found: homebridge-nest-native@latest
9 timing stage:rollbackFailedOptional Completed in 6ms
10 timing stage:runTopLevelLifecycles Completed in 1327ms
11 verbose stack Error: 404 Not Found: homebridge-nest-native@latest
11 verbose stack     at fetch.then.res (/usr/lib/node_modules/npm/node_modules/pacote/lib/fetchers/registry/fetch.js:42:19)
11 verbose stack     at tryCatcher (/usr/lib/node_modules/npm/node_modules/bluebird/js/release/util.js:16:23)
11 verbose stack     at Promise._settlePromiseFromHandler (/usr/lib/node_modules/npm/node_modules/bluebird/js/release/promise.js:512:31)
11 verbose stack     at Promise._settlePromise (/usr/lib/node_modules/npm/node_modules/bluebird/js/release/promise.js:569:18)
11 verbose stack     at Promise._settlePromise0 (/usr/lib/node_modules/npm/node_modules/bluebird/js/release/promise.js:614:10)
11 verbose stack     at Promise._settlePromises (/usr/lib/node_modules/npm/node_modules/bluebird/js/release/promise.js:693:18)
11 verbose stack     at Async._drainQueue (/usr/lib/node_modules/npm/node_modules/bluebird/js/release/async.js:133:16)

Any thoughts?

**Comment by adriancable** [comment]
Yes - it is not on npm yet. (I am probably taking over the main homebridge-nest repository, so it may eventually go there.)

For now, just clone the repository from github.
